Makefile

```diff
@@ -72,6 +72,12 @@ format:
 	@go fmt ./...
 	@golines --write-output --shorten-comments --max-len 120 .
 
+## calls --help on binary and routes output to file while ignoring dynamic fields specific
+## to indivdual builds (e.g, version)
+gen-static-help-output: eigenda-proxy
```

Comment:
```suggestion
gen-static-help-output: build
```

Reply:
should we rename the eigenda-proxy target to build? Feel its a more natural command name

Reply:
yeah that'd be a good adjustment to have